spec/system/admin_customize_form_templates_spec.rb

```diff
@@ -176,13 +175,12 @@ def quick_insertion_test(field_type, content)
       )
     end
 
-    # TODO(@keegan): Unskip this test when Upload functionality is added
-    xit "should allow quick insertion of upload field" do
+    it "should allow quick insertion of upload field" do
       quick_insertion_test(
         "upload",
         '- type: upload
   attributes:
-    file_types: "jpg, png, gif"
+    file_types: ".jpg, .png, .gif"
```

Comment:
I'm wondering if we have a test case if a file with an incorrect extension is uploaded.